migrations/1597864779562_create-items.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Item } from '../src/Item'
+
+const tableName = Item.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      name: { type: 'TEXT', notNull: true },
+      description: { type: 'TEXT' },
+      eth_address: { type: 'TEXT', notNull: true },
+      collection_id: { type: 'UUID' },
+      blockchain_item_id: { type: 'TEXT' },
+      price: { type: 'TEXT' },
+      beneficiary: { type: 'TEXT' },
+      rarity: { type: 'TEXT' },
+      type: { type: 'TEXT', notNull: true },
+      data: { type: 'JSON', notNull: true },
```

Comment:
QQ: Are we going to use `data` as a JSON then? I thought that we will use `data_id` creating different tables for each type of data.